mentees/README.md

```diff
@@ -0,0 +1,264 @@
+
+To ensure the experience is positive for everyone involved, there are a few key things you as a mentee can do to maximise your opportunity:
+
+* **Make sure you can devote yourself to the minimum required hours throughout the program.** This can vary between projects, but often you will be expected to invest a significant amount of time on a weekly basis to your work. This might also mean taking priority over your study or personal obligations, depending on the project. 
+To avoid confusion, *be realistic about how much time you can put into the program*, and communicate openly with your mentor to confirm your proposed commitment is also feasible for them.
+
+* **Be open about your skill level.** Applying for opportunities that exceed your level of skill or experience can mean more work, and possibly frustration, for those involved as the program progresses. 
+Even if you are selected for a project, it's important to have a transparent and honest conversation with your mentor(s) about the level of difficulty involved. This can help to establish a clear understanding and agreement about your ability to complete set tasks, and ensure you have the necessary resources to succeed. 
+
+*Overall, having a positive experience in any mentorship program will be dependent on good communication, diligence and taking initiative.* Your mentors and the wider community are invested in your success, so being proactive and assuming responsibility for your progress is essential. 
+
+
+## **Support Networks**
+Having a solid support network provides a safe and productive environment for mentees to seek advice and receive guidance and feedback. It can be critical in navigating challenges and obstacles, building confidence, and feeling encouraged to continue your personal and professional growth.
+
+* **Your relationship with your mentor** will be your most important connection throughout your program. In addition to providing valuable new skills and coaching, mentors can also support your professional networking and overall guidance in your career. Communicating any barriers to your learning or commitments and working together to find solutions can go a long way to developing a good working rapport.
+* **Communication channels, groups and forums** such as Slack can be a great way to find solutions to common problems you'll likely encounter on your journey. Some popular options include:
+
+[CNCF Slack channels](https://communityinviter.com/apps/cloud-native/cncf) such as **#mentoring**; where you can engage to share your knowledge to solve specific problems. 
+***Important Note:*** *It is not recommended to request general guidance or to express interest in working on issues that might be raised on these channels. Please refer to the 'What is Contribution?' section in FAQs for the preferred process.*
```

Comment:
nit: if we can provide a direct link to the "What is contribution" FAQ, let's do that.